server/constants/constants.go

```diff
@@ -144,11 +144,16 @@ const (
 
 	PipelineRequestIDApproved          = "approved"
 	PipelineRequestIDRejected          = "rejected"
+	PipelineRequestNameRun             = "run"
```

Comment:
Similar to ```PipelineRequestContextRequestName```
```suggestion
	PipelineRequestRunName             = "run"
```

Reply:
I think what we have is better. As `run` is the value of field `RequestName`